packages/@react-spectrum/s2/intl/en-US.json

```diff
@@ -17,6 +17,8 @@
   "label.(required)": "(required)",
   "label.(optional)": "(optional)",
   "menu.moreActions": "More actions",
+  "notificationbadge.plus": "{notifications}+",
+  "notificationbadge.indicatorOnly": "New activity",
```

Comment:
based off of designs suggestion. they did mention that it is very context specific but for general guidance they suggested:
- New notification
- New activity
- New update available

i just chose one of them but we can discuss which one we like best as a default label

Reply:
What you have feel like fine defaults, wonder how `+` gets localized though

Reply:
We should make sure to specifically ask about how this `+` string should be localized, and make sure it's clear for the translators.